<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VMLockClass</class>
 <widget class="QMainWindow" name="VMLockClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>394</width>
    <height>261</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VMLock</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>6</number>
    </property>
    <property name="topMargin">
     <number>6</number>
    </property>
    <property name="rightMargin">
     <number>6</number>
    </property>
    <property name="bottomMargin">
     <number>6</number>
    </property>
    <property name="spacing">
     <number>3</number>
    </property>
    <item row="0" column="1">
     <widget class="QTreeWidget" name="PETree">
      <attribute name="headerVisible">
       <bool>false</bool>
      </attribute>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QFrame" name="frame">
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string>Unique Id:</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QTextEdit" name="FunctionsEdit"/>
       </item>
       <item row="1" column="0">
        <widget class="QLineEdit" name="UIDEdit"/>
       </item>
       <item row="2" column="0">
        <widget class="QLineEdit" name="RUIDEdit"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Section Name:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLineEdit" name="SectionEdit"/>
       </item>
       <item row="7" column="0">
        <widget class="QPushButton" name="BuildButton">
         <property name="text">
          <string>Build</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Function Addresses:</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="VMLock.qrc"/>
 </resources>
 <connections/>
</ui>
